README.markdown

```diff
@@ -38,9 +44,9 @@ bootstrapped:
 
     git clone git://github.com/rspec/rspec-dev.git
     cd rspec-dev
-    bundle install --binstubs
-    bin/rake setup
-    bin/rake # runs tests in every repository
+    bundle install --path=vendor/bundle
```

Comment:
Leave it out please, it actually forces a convention onto people they may not follow, I'd actually also rather add the gem for windows users to enable compatibility with bin stubs.